src/libraries/System.Net.Http.Json/tests/FunctionalTests/System.Net.Http.Json.Functional.Tests.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
Since the implementation is ns20, you probably also wantto add a net472 configuration so that it also gets tested against .Net Framework

Reply:
Agreed. While our primary audience is Blazor (which doesn't run in NETFX) it's still a general purpose BCL API.

Reply:
I am using `LoopbackServer` for most of the tests, but it seems like it is not supported on net472. I get this error when I build against net472
```
c:\repos\runtime\src\libraries\Common\tests\System\Net\Http\GenericLoopbackServer.cs(90,30): error CS1061: 'SslProtocols' does not contain a definition for 'Tls13' and no accessible extension method 'Tls13' accepting a first argument of type 'SslProtocols' could be found (are you missing a using directive or an assembly reference?)
```

`System.Net.Http` test projects don't target netfx at all. What is the advice here to test against .net framework? Should I skip tests that use the LoopbackServer?

Reply:
I'm not sure about the System.Net.Http case, but you are effectively providing a netstandard2.0 asset which is supposed to be compatible with .NET Framework, so all tests that are applicable to .NET Framework should be run in there by having a netfx configuration. Further more, you are cross-compiling for netcoreapp as well, which means that if you only have a netcoreapp test configuration, then only the netcoreapp asset will be tested in CI which is less than ideal, netfx configuration will add coverage to the netstandard2.0 asset that goes in the package, so it is important to add that configuration. Skipping the test won't be enough as you are getting a compile-time error, not a runtime error,  so you will need to conditionally compile this test only when building for netcoreapp configuration on the .csproj, similar to what you are doing in the src project.

Reply:
@joperezr that's right, I need to make the tests compile (not skip) only against netcoreapp by ifdef-ing, but still, I would like to know how do we run client/server tests for netfx?

Reply:
https://github.com/dotnet/runtime/pull/33158 extended `LoopbackServer` to support netfx, so this is no longer an issue.